src/components/Login/Login.jsx

```diff
@@ -15,30 +15,26 @@
 // // * save user's name, token and email to the store after success login.
 // // ** TASK DESCRIPTION ** - https://d17btkcdsmqrmh.cloudfront.net/new-react-fundamentals/docs/module-3/home-task/components#login-component
 
-// import React from 'react';
+// import React from "react";
 
-// import styles from './styles.module.css';
+// import styles from "./styles.module.css";
```

Comment:
I think we should have the same quotes in all files (" " not ' '). Do you have the ability to fix it?

Reply:
fixed